pyenvbuilder/commands/create.py

```diff
@@ -71,6 +83,8 @@ def add_args(self, cmd_parser):
             '--skip-tests', action='store_true', default=False)
         cmd_parser.add_argument(
             'files', nargs='+')
+        cmd_parser.add_argument(
+            '--dest', help='Destination directory')
```

Comment:
Please let me know if i should change `--dest` to something else...

Reply:
I don't have a strong preference... as long as it includes the help and shows up at the documentation the naming is okay.